getting-started/deployment/azure-vm.mdx

```diff
@@ -52,16 +62,12 @@ You’ll need relevant permissions to create a new VM, edit its networking setti
     ```shell
     sed -i 's/http:\/\/localhost/http:\/\/'$(wget -4qO - https://ifconfig.io/ip)'/g' .env
     ```
-
-Feel free to edit the `.env` file with *vim* or another text editor according to your needs.
-
-### Install Docker and start the containers
-
-1. Install Docker using Snap:
+4. Feel free to edit the `.env` file with *vim* or another text editor according to your needs.
```

Comment:
why is it not highly recommended to change your password here? 

"4. Edit the `.env` file with **vim** or another editor. It’s highly recommended to change the default passwords."